pycbc/noise/reproduceable.py

```diff
@@ -72,7 +72,7 @@ def normal(start, end, sample_rate=16384, seed=0):
     # This is reproduceable because we used fixed seeds from known values
     block_dur = BLOCK_SAMPLES / sample_rate
     s = int(start / block_dur)
```

Comment:
@GarethDaviesGW Should not floor also be here? In fact, I think that's the immediate source of the error in the current test suite.

Reply:
yup - I realised that after seeing the error myself